src/backend/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mapeo-mobile-node-next",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
this indicates that you're using a npm version that's older than the one this project expects, hence the massive changes. you should be using npm v10 for this project.

https://stackoverflow.com/a/67874161

Reply:
Okay, I got package-lock.json from main branch.